tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -2,145 +2,84 @@
 # Licensed under the MIT License.
 
```

Comment:
Can we add top-level comments to these templates explaining what they do? Both for maintenance reasons, and I think it would make it easier to review changes to them.